corehq/apps/app_manager/templates/app_manager/partials/case_list_properties.html

```diff
@@ -53,7 +53,16 @@
                     <td data-bind="jqueryElement: $format"></td>
                 <!--/ko-->
                 <!--ko if: isTab -->
-                    <td colspan="3" data-bind="jqueryElement: header.ui"></td>
+                    <!--ko if: !$data.hasNodeset -->
```

Comment:
Using `ko ifnot: ` would make this more clear that its an `if else` type structure.